### ISSUE TITLE
MRTK compatibility with Pimax VR headsets

### ISSUE THREAD

**Issue by jasonhbartlett** [microsoft/MixedRealityToolkit-Unity#3364]
Question: Does anybody know if MRTK is compatible (can be used in harmony with) the Pimax VR headset?  I do not yet have a PimaxVR headset, but was considering ordering one of the latest 8K model given it is now available for $899.00 [Pimax 8K Headset](https://pimaxvr.com/products/pimax-8k-series?variant=19912761606203)

Their website includes a developer page with downloadable Unity plugin SDK which I have briefly looked at but wondered if the MRTK would have any conflicts with that?
[Pymax VR developer](https://pimaxvr.com/pages/developer) 

Basically as I don't have any experience developing for other VR platforms other than the UWP Mixed Reality headsets, I'm looking for some guidance  before I buy.  With the Pimax will I be able to build and run a VR app that runs native on Windows 10 (without Steam)?  Will I be able to modify the app I've already developed for a Samsung Odyssey to work now with the Pimax?  Any advice is appreciated.

**Comment by SimonDarksideJ** [comment]
I'll have a look at it, however when last I checked, PiMax supports OpenVR, so you can run it under that for now.
Looking at the controllers, we'll not likely need dedicated support, but we'll see.

**Comment by jasonhbartlett** [comment]
Okay. Just did a quick test to see could take my MR VR app that currently targeted UWP and instead change it to Standalone Desktop with select Virtual Reality Supported in the Player Settings and a target device of OpenVR. 
(I don't have a Pimax yet, so I'm just testing with my Samsung Odyssey HMD)
Basically, it worked.  I did have to import the SteamVR plugin asset from the Unity asset store, and I have to run my app in "Windows Mixed Reality for SteamVR". So I DO need Steam.  Oh well, that's fine.  I just need to figure out how to launch my app from within the SteamVR home space.  Whatever they call that.  Rather than have to find the .exe in Windows explorer.
Regarding my project... currently my App only uses the MRTK OnFocusEnter and OnFocusExit inputs and they work fine.  I don't use any controllers so I can't say if they have any conflicts or not.  But based on this, with OpenVR I should be able to run my app on the Pimax.  But next test is to just import the Pimax SDK into the project as well.
